Network/src/Parsing/parse_args.c

```diff
@@ -42,16 +47,18 @@ void do_parsing_argument(parsing_argument_t data, int *i, int j)
         *data.n_flag = (j == NAMES);
     }
     free(tmp);
+    return false;
 }
 
-void check_all_flags(parsing_argument_t data, int *i)
+bool check_all_flags(parsing_argument_t data, int *i)
 {
     for (int j = 0; j < ALL_FLAGS; j++) {
```

Comment:
variable has to be defined at the top of the function

```suggestion
    int j;

    for (j = 0; j < ALL_FLAGS; j++) {
```

Reply:
still no